### DISCUSSION TITLE
Do we need to update CefSharp 75.1.143.0?

### DISCUSSION THREAD

**Discussion by krubin723** [cefsharp/CefSharp#4615] [Q&A]
Hi,

We currently use and older 75.1.143.0 CefSharp. 
We are using the modern Chrome and Firefox Browsers.
We have an application that has a chat feature and it accesses a url and related a Javascript on a remote site.
Is it possible that we need to update our version of CefSharp to work with the new Javascript?

Thank you,
Ken

**Comment by amaitland** [comment]
> We currently use and older 75.1.143.0 CefSharp.

Using such an old version on the internet is a security risk.

> Is it possible that we need to update our version of CefSharp to work with the new Javascript?

Yes it's possible.

You can test out the latest version using https://github.com/cefsharp/CefSharp.MinimalExample
